frontend/encore/dev-server.rst

```diff
@@ -65,29 +65,30 @@ the ``package.json`` file:
 
 .. code-block:: diff
 
-    {
```

Comment:
first line comment is missing

Reply:
Yeah, I wanted to test whether `:dedent: 0` was needed here. And seems like Sphinx runs fine without it:

![image](https://user-images.githubusercontent.com/749025/108698495-7cdb5500-7504-11eb-9f94-841ac7bfd735.png)

So I think we should probably update all code blocks to just follow normal identation when there is no file line comment?